packages/lesswrong/components/posts/dialogues/DialogueSubmit.tsx

```diff
@@ -0,0 +1,153 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Components, registerComponent, getSiteUrl } from '../../../lib/vulcan-lib';
+import Button from '@material-ui/core/Button';
+import classNames from 'classnames';
+import { useCurrentUser } from "../../common/withUser";
+import { useTracking } from "../../../lib/analyticsEvents";
+import {forumTitleSetting, forumTypeSetting, isLW} from "../../../lib/instanceSettings";
+import { forumSelect } from '../../../lib/forumTypeUtils';
+
+export const styles = (theme: ThemeType): JssStyles => ({
+  formButton: {
+    fontFamily: theme.typography.commentStyle.fontFamily,
+    fontSize: isEAForum ? 14 : 16,
+    marginLeft: 5,
+    ...(isEAForum ? {
+      textTransform: 'none',
+    } : {
+      paddingBottom: 4,
+      fontWeight: 500,
+      "&:hover": {
+        background: theme.palette.buttons.hoverGrayHighlight,
+      }
+    })
+  },
+  secondaryButton: {
+    ...(isEAForum ? {
+      color: theme.palette.grey[680],
+      padding: '8px 12px'
+    } : {
+      color: theme.palette.text.dim40,
+    })
+  },
+  submitButtons: {
+    marginLeft: 'auto'
+  },
+  submitButton: {
+    ...(isEAForum ? {
+      backgroundColor: theme.palette.buttons.alwaysPrimary,
+      color: theme.palette.text.alwaysWhite,
+      boxShadow: 'none',
+      marginLeft: 10,
+    } : {
+      color: theme.palette.secondary.main
+    })
+  },
+  cancelButton: {
+  },
+  draft: {
+  },
+  feedback: {
+  }
+});
+
+const isEAForum = forumTypeSetting.get() === "EAForum"
```

Comment:
There's a global we should import instead of redefining it here

Reply:
Fixed